src/components/LandAssignENSPage/LandAssignENSForm/LandAssignENSForm.tsx

```diff
@@ -73,34 +74,36 @@ export default class LandAssignENSForm extends React.PureComponent<Props> {
             {t('land_assign_ens_page.set_name_message', { strong: (children: React.ReactElement) => <strong>{children}</strong> })}
           </p>
         </Row>
-        <Row>
-          <div className={isReclaimButtonDisabled ? 'box box-disabled' : 'box'}>
-            <h3>{t('land_assign_ens_page.reclaim')}</h3>
-            <div className="message-box">
-              <p>{t('land_assign_ens_page.reclaim_explanation')}</p>
-              <NetworkButton
-                type="submit"
-                disabled={isReclaimButtonDisabled}
-                onClick={this.handleReclaim}
-                className={reclaimContentButtonClassName}
-                loading={isWaitingTxReclaim}
-                primary
-                network={Network.ETHEREUM}
-              >
-                {hasReclaimError ? (
-                  t('global.retry_tx')
-                ) : !needsReclaiming ? (
-                  <>
-                    {t('global.approved_tx')}
-                    {!isWaitingTxReclaim ? <Icon name="check" /> : null}
-                  </>
-                ) : (
-                  t('global.submit')
-                )}
-              </NetworkButton>
+        {!isEnsAddressEnabled ? (
```

Comment:
Isn't this removing the whole reclaim process if the `isEnsAddress` feature is not enabled?

Reply:
when it is enabled the user should never reach this page without reclaiming the name before.. its blocking the assign address and land buttons on the list and on the detail page